### ISSUE TITLE
Issue: Script Helper Tool with the same experiment name overwrites previous files without warning

### ISSUE THREAD

**Issue by A-Ivan** [BioMotionLab/TUX#44]
If you use the Script Helper Tool to setup the experiment automatically more than once, with the same name, it will overwrite any previously created and edited files.

I suggest adding a function to check if these files already exist and to provide a popup of some kind asking the user to confirm if they want to overwrite the existing files. 

Confirming would recreate/overwrite the files experiment files.

Canceling would keep the user at the Script Helper window without overwriting any files. The user could then change the experiment name and then create/setup the desired files.

**Comment by AdamBebko** [comment]
I agree this would be a useful check. Unfortunately, I don't have the resources at this time to do the work. Please feel free to submit a pull request, and I can take a look!
